internal/system/agent/router.go

```diff
@@ -99,7 +115,7 @@ func operationHandler(w http.ResponseWriter, r *http.Request) {
 
 func configHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
-	LoggingClient.Debug("service names: ", vars)
+	LoggingClient.Debug("retrieved service names","names", vars)
```

Comment:
Here also, and elsewhere, please avoid multiple keys. Collapse them as indicated in the earlier comment 👍